wizeline/project-lab
# chore: added archived migration, and actives filter

#### What does this PR do?

-Added the necessary logic to filter the project search page by their IsArchived Project.
-Added migration to create the field "IsArchived" at the projects table.

#### Where should the reviewer start?

At the file app/pages/projects/search.tsx

#### How should this be manually tested?

-Go to the search Proposals / My Proposals page
-Verify that the "Active" is selected by default
-Crear the "Active" filter
-Verify that all projects are displayed
-Click on my proposals
-Verify that the Active filter is selected by default. 

#### Any background context you want to provide?

This is my first feature and I created a migration, so is possible that could mess with the current code.

#### What are the relevant tickets?

Ticket [#185](https://github.com/wizeline/project-lab/issues/185l)

#### Screenshots

![image](https://user-images.githubusercontent.com/97201781/155799033-2a730c4f-6c27-4289-ad3c-25d47d17ab07.png)

#### Questions

I had to add a flag, because of the eslint rule eslint-plugin-react-hooks/exhaustive-deps, if you could advice me a better approach please let me know. As always, feedback is very welcomed. 

#### Checklist
